dpgen/generator/run.py

```diff
@@ -3026,7 +3029,7 @@ def post_fp_cp2k (iter_index,
         all_sys = None
         for oo in sys_output :
             _sys = dpdata.LabeledSystem(oo, fmt = 'cp2k/output')
-            _sys.check_type_map(type_map = jdata['type_map'])
+            #_sys.check_type_map(type_map = jdata['type_map'])
```

Comment:
Could you explain why the check_type_map is commented out?

Reply:
Thanks for your comment, I have edited my comment.